### PR TITLE
test: add an integration test about pd client freeze

### PR DESCRIPTION
**Pull request by leoppro**
### What problem does this PR solve? 
add an integration test about pd client freeze

### Check List 

Tests 

 - Unit test
 - Integration test

### Release note

- No release note

### CODE REVIEW

File: tests/pd_freeze/run.sh [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    # record tso before we create tables to skip the system table DDLs
+    start_ts=$(run_cdc_cli tso query --pd=http://$UP_PD_HOST_1:$UP_PD_PORT_1)
+
+    run_sql "CREATE table test.simple1(id int primary key, val int);"
+    run_sql "CREATE table test.simple2(id int primary key, val int);"
+    
+    export GO_FAILPOINTS='github.com/pingcap/ticdc/cdc/captureServerRecovered=1*return(true);github.com/pingcap/ticdc/cdc/captureGetCurrentVersionAfterServerRecovered=return(true)'
+    run_cdc_server --workdir $WORK_DIR --binary $CDC_BINARY
+
+    TOPIC_NAME="ticdc-simple-test-$RANDOM"
```
Comment (leoppro) [inline tests/pd_freeze/run.sh:27]:
```suggestion
    TOPIC_NAME="ticdc-pd-freeze-test-$RANDOM"
```

**Comment by hi-rustin** [comment]
@leoppro PR conflict, can you please resolve the conflict? Thanks!

**Pull request closed without merging**